diff --git a/dist/src/index.js b/dist/src/index.js
index 19f50fe927f769a2c3c917cc07dc51e702d572dc..178485f64436f4ac9c6212f2e3dfced8645c7126 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -86892,7 +86892,7 @@ var InspectorProxyController = class {
     const server = await this.#server;
     server.closeAllConnections();
     return new Promise((resolve6, reject) => {
-      server.close((err) => err ? reject(err) : resolve6());
+      server.close((err) => err && err.code !== "ERR_SERVER_NOT_RUNNING" ? reject(err) : resolve6());
     });
   }
 };
