diff --git a/dist/src/index.js b/dist/src/index.js
index 38cb0eebc9fde146fefc5af79282af2daef6c492..e2e0ef1018bfd5c729de4550f0d8a9b7750c5959 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -87016,7 +87016,7 @@ var InspectorProxyController = class {
     const server = await this.#server;
     server.closeAllConnections();
     return new Promise((resolve6, reject) => {
-      server.close((err) => err ? reject(err) : resolve6());
+      server.close((err) => err && err.code !== "ERR_SERVER_NOT_RUNNING" ? reject(err) : resolve6());
     });
   }
 };
